cypress/integration/homeview_spec.js

```diff
@@ -1,14 +1,9 @@
 describe('VegShare, main page view', () => {
-  const baseUrl = 'https://veg-share.herokuapp.com'
   beforeEach(() => {
-    cy.visit(baseUrl)
+    cy.visit('/')
```

Comment:
I see that _this is working_, and that when it runs these tests automatically, it will just be checking that base path `/` on the URL for the deployed site (now Vercel, not Heroku) - but I'm curious how this works when the declared URL to visit is just the '/'...? I would have guessed this would still need to look more like the previous version, just updated with the Vercel deployed url.

Reply:
This is related to the cypress.json file in the top directory. Inside there is where the baseURL was changed to localhost:3000 for testing purposes. At least that's how I understand it! This was from the guide that I followed.